.stickler.yml

```diff
@@ -0,0 +1,3 @@
+---
+  linters:
+    stylelint:
```

Comment:
- This not the linter configuration that you should adapt. Check Microverse's [repo](https://github.com/microverseinc/linters-config/tree/master/css) and follow the instruction.

Reply:
Yes, in fact we usa a css gradient tool , and i copied the resulting gradient code, but we did as you advised, and all good and ready to go TY!